src/client/components/pages/auth/styles.scss

```diff
@@ -47,3 +47,4 @@
     text-align: center;
   }
 }
+}
```

Comment:
here also, format and sort on every module you work.

`option+shift+f` and `option+shift+o`